jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TargetImageParameters.java

```diff
@@ -60,12 +61,19 @@ public void setImage(String image) {
   @Input
   @Optional
   public Set<String> getTags() {
-    if (System.getProperty(PropertyNames.TO_TAGS) != null) {
-      return ImmutableSet.copyOf(
-          ConfigurationPropertyValidator.parseListProperty(
-              System.getProperty(PropertyNames.TO_TAGS)));
+    String property = System.getProperty(PropertyNames.TO_TAGS);
+    Set<String> tagsValue =
+        property != null
+            ? ImmutableSet.copyOf(ConfigurationPropertyValidator.parseListProperty(property))
+            : tags;
+    String source = property != null ? PropertyNames.TO_TAGS : "jib.to.tags";
+    if (tagsValue.stream().anyMatch(Strings::isNullOrEmpty)) {
+      throw new IllegalArgumentException(source + " has empty tag");
     }
-    return tags;
+    if (tagsValue.stream().anyMatch(str -> str.contains(" "))) {
```

Comment:
What I meant was checking for something like `"   "` but maybe jib-core already validates these values before going?
Perhaps your original condition was enough, lemme see if jib core validates this.

Reply:
Sorry, yeah so looks like we handle this properly already at the processing level: https://github.com/GoogleContainerTools/jib/blob/master/jib-core/src/main/java/com/google/cloud/tools/jib/api/ImageReference.java#L68
so `isNullOrEmpty` should be sufficient?